confgenerator/testdata/invalid/linux/logging-receiver_tcp_type_missing_required_parameter_format/golden_error

```diff
@@ -0,0 +1,8 @@
+[3:13] "format" is a required field
```

Comment:
I prefer the error you get if you don't mark this `required` (`"format" must be one of [json]`) but I don't feel strongly about it

Reply:
Custom error messages from the validator can be a future PR. Leaving as is for consistency.